### DISCUSSION TITLE
Extra properties on returned state for Popover

### DISCUSSION THREAD

**Discussion by SiTaggart** [ariakit/ariakit#889] [Questions] [answered]
Quick question. We've noticed that for Popover there are some extra undocumented properties that are return fro the state hook. Most notably the `show` method.

We notice the `hide` method is listed, but not `show`. It works as expected though. 

Wondering if the return is wrong or the docs are wrong?

**Comment by diegohaz** [comment, marked as answer]
The properties shown in the documentation are the ones required by the respective components (`Popover` requires `hide`, but not necessarily `show`). We still don't have docs on the site for the returned object of state hooks, but they do have JSDocs so you can probably access their docs in the code editor.

You can use them safely unless they're prefixed with `unstable_`.

**Reply by SiTaggart** [reply]
OK perfect, thank you!
